primer/test/Tests/Action/Prog.hs

```diff
@@ -873,6 +873,27 @@ unit_copy_paste_import =
         Left err -> assertFailure $ show err
         Right assertion -> assertion
 
+unit_DeleteTypeDef :: Assertion
+unit_DeleteTypeDef = progActionTest
+  ( defaultProgEditableTypeDefs
+      $ sequence
+        [ do
+            x <- emptyHole `ann` (tcon tT `tapp` tcon (tcn "Bool") `tapp` tEmptyHole)
+            astDef "def" x <$> tEmptyHole
+        ]
```

Comment:
Might be better to also use the type's constructors at term level, both as constructors and as patterns in a `case`